### DISCUSSION TITLE
Player lock for mobile

### DISCUSSION THREAD

**Discussion by Travillion** [advplyr/audiobookshelf#2237] [Ideas]
I think it would be helpful if there was a way to "lock" the player from receiving any commands, similar to how VLC player on Android has a lock.

I went for a bike ride this morning and, despite me turning my  phone screen off, I had so many seeks initiated by my pocket. Thankfully I could get back to where I was with the player history, but it was still really frustrating. I assumed they were pocket seeks. It never happened while I was actively listening. I would pause it, interact with my kids, and when I would play again, I would be hours behind. The play history showed seeks.

**Comment by nichwall** [comment]
I just tried to find the lock in VLC, but couldn't find the setting.

Disabling seek from the notification has been mentioned a few times (could only find the iOS related feature request)
https://github.com/advplyr/audiobookshelf-app/issues/526

You could disable the notification for the app through system settings in the meantime.

**Comment by Travillion** [comment]
I'll try and take a screenshot from my kid's tablet. It's basically a lock button that disables all the other media controls until it's been unlocked. It's nice to know I can do that from a system level in the meantime, though I don't think that's a good solution long-term as it requires several steps. Most of the time I don't need this because I'm driving in my car or sitting at my desk. It's only when I'm out on a bike ride or doing something physical.

**Reply by nichwall** [reply]
Yeah, I agree the system level setting isn't ideal, but it's something in the meantime.
